src/bson/bson-config.h

```diff
@@ -157,4 +162,14 @@
 # undef BSON_HAVE_DECIMAL128
 #endif
 
+#define BSON_HAVE_RAND_R 1
+#if BSON_HAVE_RAND_R != 1
+# undef BSON_HAVE_RAND_R
+#endif
+
+#define BSON_HAVE_STRLCPY 1
```

Comment:
Is strlcpy available on manylinux1? Windows?

Reply:
I believe that strcpy is available on linux, but is deprecated on Windows.